app/assets/javascripts/calendars/holidays.es6

```diff
@@ -11,23 +11,15 @@
         eventBorderColor: '#000',
         events: el.data('holidays-path'),
         header: {
-          center: 'title',
-          right: 'createHoliday month,agendaWeek,agendaDay today prev,next'
+          right: 'month, agendaWeek, agendaDay today prev,next'
         },
-        customButtons: {
-          createHoliday: {
-            text: 'Create holiday',
-            click: () => {
-              window.location = this.$el.data('new-holiday-path');
-            }
-          }
-        },
-        eventRender: (event, element) => {
-          var $button = element.append(
-            '<button class="close t-delete-holiday">' +
-            '<span aria-hidden="true">X</span><span class="sr-only">Remove slot</span>' +
-            '</button>'
-          );
+        eventRender: (event, element, view) => {
+          const $button = $(`
+            <button class="close t-delete-holiday">
+              <span aria-hidden="true">X</span><span class="sr-only">Remove slot</span>
```

Comment:
I'm thinking to leave this for now, as we are about to remove this way to delete things anyway - something to apply to the slot picker calendar though.